PKGBUILD

```diff
@@ -3,32 +3,50 @@
 
 pkgname=linx-server
 pkgver=2.3.5
-pkgrel=1
-pkgdesc='Self-hosted file/code/media sharing website '
-arch=('i686' 'x86_64' 'arm')
```

Comment:
Why are you removing those architectures?

Reply:
Sorry, I misread the PKGBUILD docs.